README.md

```diff
@@ -4,6 +4,15 @@ A project based learning activity for people who are getting started with Git an
 
 You can play the game at: https://githubschool.github.io/github-games/
 
+### Instructions for playing the game
+
+1. Press the space bar to begin.
```

Comment:
why cmd said 'git' is not recognized as an internal or external command,
operable program or batch file ?